### DISCUSSION TITLE
Raven Replication - Error -  Stream was not readable

### DISCUSSION THREAD

**Discussion by santoshchan** [ravendb/ravendb#15744] [General]
Hi,
 
 We are moving the Raven DB from Classic VM's to Azure ARM VM ,  we have a replication setup between primary and secondary raven servers and vice-versa in case of failures
We are noticing more conflicts reported in the secondary ARM VM, post migration
On investigating we could see  in the logs  an error message - ' Raven.Bundles.Replication.Tasks.ReplicationTask,  Error, Could not replicate to http://ip.address./databases/xxxx  System.ArgumentException: Stream was not readable.'

Raven version used is 3.5

**Comment by ayende** [comment]
Please note, that version is not supported unless you have an extended support contract.
In that case, please open a ticket using the support mechanism.
